api/tableau_api/views.py

```diff
@@ -8,9 +69,180 @@ class SummaryStatisticDailyStudyView(TableauApiView):
     path = '/api/v0/studies/<string:study_id>/summary-statistics/daily'
 
     def get(self, study_id):
-        # Todo (CD): Implement SummaryStatisticDaily database model
-        # Todo (CD): Implement this logic
-        return 'ok'
+        request.values = dict(request.values)
+        request.values['study_id'] = study_id
+        errors, query = self._validate_query(**request.values)
+        if errors:
+            return self._render_errs(errors)
+        queryset = self._query_database(**query)
+        # alternative approach?
+
+        class SummaryStatisticDailySerializer(serializers.ModelSerializer):
+            class Meta:
+                model = SummaryStatisticDaily
+                fields = query["fields"]
+            if "participant" in query["fields"]:
+                participant = serializers.SlugRelatedField(slug_field="patient_id", read_only=True)
+            if "study" in query["fields"]:
+                study = serializers.SlugRelatedField(slug_field="object_id", read_only=True)
+
+        serializer = SummaryStatisticDailySerializer(queryset, many=True)
+        #  the fact that this is an error (I dont mean False) is genuinely upsetting: print(serializer.is_valid())
+        return JSONRenderer().render(serializer.data)
+
+    @staticmethod
+    def _query_database(study_id, end_date, start_date, limit, ordered_by, order_direction, participant_ids, fields):
```

Comment:
Let's collapse the default values into here instead of being in a method of the `ApiQueryForm`.